share/src/main/resources/alfresco/site-webscripts/org/alfresco/components/form/control-wrapper.post_pt_BR.properties

```diff
@@ -46,7 +46,7 @@ form.control.date-picker.entry.datetime.format.nojs=YYYY-MM-DDThh:mm:ss.sTZD
 
 ## From and To labels for date range control
 form.control.date-range.from=De
-form.control.date-range.to=At\u00e9
+form.control.date-range.to=Para
```

Comment:
You repeating the same error again... In Portuguese, we do not use De... Para when we are talking about date ranges.
The original text is right, De... Até...

De... Para is used when you are moving things, from somewhere to another place.